src/blt_mad/btlMad_test.go

```diff
@@ -0,0 +1,40 @@
+package blt_mad
+
+import "testing"
+
+//figure out how to test these
+func TestBltMad10(t *testing.T) {
+	tau := 10.0
+	// Test case 1: all zero
+	data1 := []float64{0.0, 0.0, 0.0, 0.0}
+	expected1 := []float64{}
+	result1 := BltMad(data1, tau)
+	if !equalSlices(expected1, result1) {
+		t.Errorf("Test case 1 failed. Got: %f, Expected: %f", result1, expected1)
+	}
+
+	//Test case 2: bgp data 1
```

Comment:
similar to `mad_test.go`, it can be good to break these up into multiple functions